modules/tinymce/src/themes/silver/main/ts/ui/icons/Icons.ts

```diff
@@ -17,8 +19,32 @@ const getOr = (name: string, icons: IconProvider, fallback: Optional<string>): s
 
 const getFirst = (names: string[], icons: IconProvider): string => Arr.findMap(names, (name) => Optional.from(icons()[name.toLowerCase()])).getOrThunk(() => defaultIcon(icons));
 
+const render = (tagName: string, iconHtml: string, classes: string[], behavioursOpt: Optional<Behaviour.AlloyBehaviourRecord>): SimpleOrSketchSpec => {
+  const iconSpec = {
+    dom: {
+      tag: tagName,
+      innerHtml: iconHtml,
+      classes
+    },
+    events: AlloyEvents.derive([
+      AlloyEvents.runOnAttached((comp) => {
+        // set focusable=false on SVGs to prevent IE 11 from focusing the toolbar when tabbing into the editor
+        SelectorFind.child(comp.element, 'svg').each((svg) => Attribute.set(svg, 'focusable', 'false'));
+      })
+    ])
+  };
+
+  const behavioursSpec = behavioursOpt.map((behaviours) => ({ behaviours })).getOr({});
+
+  return {
+    ...iconSpec,
+    ...behavioursSpec
+  };
+};
```

Comment:
This is done this way to prevent `behaviours` from being overwritten during the spread, but in its current state it looks very "refactorable" -- that is, someone might simplify this later and just return behaviours in this response like `behaviours: behavioursOpt.getOr({})`. Was there a specific reason you removed the `fold` from earlier?

Reply:
I asked Diogo to remove the fold, as it's a bad pattern to be using for this case as it prevents multiple optional arguments being folded into the spec (even if we only have one now).

Reply:
I should also note that this pattern is what's used in other places in Silver. You certainly could do `behaviours: behavioursOpt.getOr({})` though, as that would almost be identical (the current one means the key/value isn't included at all, while the latter includes an empty object).